src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/IncrementalBuildTest.cs

```diff
@@ -342,7 +342,6 @@ public void AppProjectTargetsDoNotBreak ()
 					Path.Combine (intermediate, "build.props"),
 					Path.Combine (intermediate, proj.ProjectName + ".dll"),
 					Path.Combine (intermediate, "android", "assets", proj.ProjectName + ".dll"),
-					Path.Combine (Root, b.ProjectDirectory, "packages.config"),
```

Comment:
I think I saw a test failure here, you probably need to list `project.assets.json` here for the test to touch, so it can check the `_CleanIntermediateIfNuGetsChange` target builds incrementally.